### ISSUE TITLE
Focus Tab traversal - keyboard accessibility

### ISSUE THREAD

**Issue by pgrzywa92** [aleksanderwozniak/table_calendar#820]
For now, I cannot implement a full keyboard accessibility application because table_calendar does not support a focus system, especially a tab traversal policy.
Now when I press the tab key on the keyboard I can pass through my widgets but the calendar are skipped because it doesn't support the focus system.

It would be great if you would implement tab traversal and focus policy in your features. Regarding to Flutter rules:
[Understanding Flutter's keyboard focus system](https://docs.flutter.dev/ui/interactivity/focus)

**Comment by niteshsh4rma** [comment]
Any work on this?

**Comment by arctouch-celiogonzaga** [comment]
Is there any news?
